tests/unit_tests/test_suassystem_sda_converter.py

```diff
@@ -42,16 +64,13 @@ def test_reset_obstacles(self):
 
         self.assertEqual(obstacle_map.obstacles.size, 0)
 
-    def test_set_waypoints(self):
-        """
-        Test the set_waypoints() method
-        """
-        test_waypoint = Location(38.8742103, -77.3217697, 91.44000244140625)
-        self.sda_converter.set_waypoint(test_waypoint)
-        obstacle_map = self.sda_converter.obstacle_map
-        waypoint_holder = obstacle_map.drone.get_waypoint_holder()
-        new_gps_point = inverse_haversine(self.sda_converter.initial_coordinates,
-            [waypoint_holder[0][0], waypoint_holder[0][1], 91.44000244140625])
+    def test_set_uav_position(self):
+        new_location = Location (40, -80, 95)
+        self.sda_converter.set_uav_position(new_location)
 
-        self.assertTrue(abs(new_gps_point.get_lat() - test_waypoint.get_lat()) < 0.004)
-        self.assertTrue(abs(new_gps_point.get_lon() - test_waypoint.get_lon()) < 0.004)
+        converted_uav_location = convert_to_point(self.initial_coordinates, new_location)
+
+
+        self.assertEqual(self.sda_converter.obstacle_map.drone.point[0], converted_uav_location[0])
+        self.assertEqual(self.sda_converter.obstacle_map.drone.point[1], converted_uav_location[1])
+        self.assertEqual(self.sda_converter.obstacle_map.drone.point[2], converted_uav_location[2])
```

Comment:
If these are both Numpy arrays, you could use an array_equal compare method here. Here is the documentation for the method: https://docs.scipy.org/doc/numpy/reference/generated/numpy.array_equal.html